### ISSUE TITLE
GPU Crash

### ISSUE THREAD

**Issue by AndreDK7** [cy33hc/vita-ftp-client#1]
I had a GPU Crash while i was navigating between my folders into my FTP Server.

**Comment by cy33hc** [comment]
@AndreDK7 What ftp server are you using? Also any special character in your directories and does it occur on the same directory always?

**Comment by AndreDK7** [comment]
I was using my phone as FTP Server with MiX file explorer application. The folders doesn't have any special characters. I was fearing another GPU Crash before testing with another FTP Server.

Edit.: just noticed a new version. I'll test with v1.12 here to see what happens.

**Comment by cy33hc** [comment]
@AndreDK7 I just release 1.13 with a potential crash fix. Can you try that one out. And do you have a link to the app?

**Comment by AndreDK7** [comment]
Yeah, i saw that when i installed 1.12 and tried to upgrade from inside the app. I did a overclock with PSVShell just to make sure before testing again with 1.13 and didn't had any gpu crash (also doing a overclock could be a solution here, cuz even the PS Button menu navigation was more faster than before). The only issue that i had was while server changing. I got a error and then FTP Client closed with a error log from Vita. My workaround for that was closing and then open the application before connecting to another ftp server.

Edit: here's a link for MiX file explorer: https://forum.xda-developers.com/attachments/mixplorer_v6-58-4_b22020910-apk.5532511/
